lib/datasets/afinn.rb

```diff
@@ -0,0 +1,45 @@
+require "csv"
+require "zip"
+require_relative "dataset"
+#require_relative "zip_extractor"
+
+module Datasets
+  class Afinn < Dataset
```

Comment:
Could you keep upper case such as "HTTP" not "Http" for "HTTP"?

```suggestion
  class AFINN < Dataset
```